src/components/cookie-message/style.less

```diff
@@ -1,30 +1,58 @@
 :host {
```

Comment:
:host is the component itself, moving everything from here will cause the component to stop working